source/FreeplayState.hx

```diff
@@ -320,6 +320,7 @@ class FreeplayState extends MusicBeatState
 
 				FlxG.sound.list.add(vocals);
 				FlxG.sound.playMusic(Paths.inst(PlayState.SONG.song), 0.7);
+				FlxG.sound.list.add(vocals);
```

Comment:
What's the point if it's added literally right before the music plays?

Reply:
it doesnt actually play the vocals if its before the music thought

Reply:
vocals get played literally right after the playmusic command

https://github.com/ShadowMario/FNF-PsychEngine/blob/743898050850c2244cac2dae653681668e4f47a8/source/FreeplayState.hx#L322-L323

Reply:
well for me it does